.devcontainer/post-start-command.sh

```
@@ -1 +1 @@
-corepack enable && corepack prepare pnpm --activate
\ No newline at end of file
+corepack enable && corepack prepare pnpm@10.15.1 --activate
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)